mteb/models/voyage_models.py

```diff
@@ -199,8 +199,8 @@ def _batched_encode(
     embed_dim=1024,
     open_source=False,
 )
-
-voyage_multilingual_2 = ModelMeta(  # reference: https://blog.voyageai.com/2024/06/10/voyage-multilingual-2-multilingual-embedding-model/"
+# see https://blog.voyageai.com/2024/06/10/voyage-multilingual-2-multilingual-embedding-model/"
```

Comment:
There is a reference key in ModelMeta would move it there

Reply:
Hm I don't see one: https://github.com/embeddings-benchmark/mteb/blob/700a41d274189f3214ae7551d664f76192b0f49c/mteb/model_meta.py#L63 - which one is it?